src/store/ChartStore.js

```diff
@@ -293,6 +294,14 @@ class ChartStore {
         }
     }
 
+    setIsMobile(status) {
+        this.isMobile = status;
+
+        if (this.isMobile && this.rootNode) {
+            this.rootNode.setAttribute('class', 'smartcharts-mobile');
+        }
```

Comment:
Set the className inside [Chart.jsx](https://github.com/binary-com/SmartCharts/blob/dev/src/components/Chart.jsx#L47)

```jsx
<cq-context ref={...} class={isMobile ? 'is-mobile' ? ''}> ... </cq-context>
```